mapswipe_workers/python_scripts/results_to_mapping_sessions_results.py

```diff
@@ -0,0 +1,66 @@
+import sys
+
+import pandas as pd
+
+from mapswipe_workers import auth
+from mapswipe_workers.definitions import logger
+
+
+def copy_results_batch(first_timestamp, last_timestamp):
+    logger.info(
+        f"Start process for : ms.start_time>={first_timestamp} "
+        f"and ms.start_time<{last_timestamp}"
+    )
+    p_con = auth.postgresDB()
+    query = """
+        -- create table with results for given time span
+        drop table if exists results_batch;
+        create table results_batch as
+        select r.*
+        from mapping_sessions ms, results r
+        where
+            ms.start_time >= %(first_timestamp)s
+            and ms.start_time < %(last_timestamp)s
+            and ms.project_id = r.project_id
+            and ms.group_id = r.group_id
+            and ms.user_id = r.user_id;
+        insert into mapping_sessions_results
+        (
+        select
+            m.mapping_session_id
+            ,r.task_id
+            ,r."result"
+        from results_batch r, mapping_sessions m
+        where
+            r.project_id = m.project_id and
+            r.group_id = m.group_id and
+            r.user_id = m.user_id
+        )
```

Comment:
I think this can go away entirely if you pick up the session id above. Save a bit of work.